gridt/tests/controllers/test_subscription.py

```diff
@@ -0,0 +1,156 @@
+from ..basetest import BaseTest
+from unittest import skip
+
+from gridt.controllers.subscription import (
+    _get_subscription,
+    is_subscribed,
+    get_subscribers,
+    get_subscriptions,
+    new_subscription,
+    remove_subscription
+)
+import gridt.exc as E
+from gridt.models.subscription import Subscription
+
+
+class SubscriptionControllerUnitTest(BaseTest):
+    
+    def test_helper_get_subscription(self):
+        user = self.create_user()
+        self.session.commit()
+        movement = self.create_movement()
+
+        user_id = user.id
+        movement_id = movement.id
+
+        # Test when subcription does not exist that an error is raised
+        with self.assertRaises(E.SubscriptionNotFoundError):
+            _get_subscription(user_id, movement_id, self.session)
+
+        subscription1 = Subscription(user, movement)
+        self.session.add(subscription1)
+
+        subscription2 = _get_subscription(user.id, movement.id, self.session)
+        self.assertEqual(subscription1.id, subscription2.id)
+        self.assertEqual(subscription1.user_id, subscription2.user_id)
+        self.assertEqual(subscription1.movement_id, subscription2.movement_id)
+
+    def test_is_subscribed(self):
+        user = self.create_user()
+        movement = self.create_movement() 
+        self.session.commit()
+
+        user_id = user.id
+        movement_id = movement.id
```

Comment:
I see that some code is run for a lot of the tests, is it not possible in python to have a setup function that runs before each tests? That way the code is a bit more reusable and shorter.

Reply:
It is possible. I was probably a bit too lazy with tests